test/sagemaker_endpoint_tests/tensorflow/test_endpoint.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
looks like you have a number of unused imports, etc in this file. I recommend running pylint on this to take care of some of these issues

Reply:
Yes, they will be removed.